docs/_config.yml

```diff
@@ -60,13 +60,13 @@ categoryInfo:
     cn: 用户指南
```

Comment:
oh can you also remove the `# TODO:` line above this?

Reply:
Removed